pandas/core/arrays/arrow/array.py

```diff
@@ -2644,7 +2644,11 @@ def _dt_minutes(self) -> Self:
     def _dt_seconds(self) -> Self:
         return type(self)(
             pa.array(
-                self._to_timedeltaarray().seconds, from_pandas=True, type=pa.int32()
+                [
```

Comment:
Unfortunately this is not where we want to fix things - doing a Python loop here would be a significant slow down compared to the non-arrow case.

I think the problem is that the Arrow duration does not have the same storage as the CPython timedelta, yet we use the CPython API to try and extract those components in pandas/_libs/tslibs/timedeltas.pyx

I vaguely recall this being a conversation with the larger team before but can't remember specifics. Maybe @jbrockmendel or @jorisvandenbossche knows

Reply:
this looks like it matches what we get for the non-pyarrow .dt.seconds, so i dont think we'd want to change it.  only in .dt.components

Reply:
Sorry. I thought that since a loop was already being done for the other properties (hours, minutes and milliseconds) then it would be reasonable to do it for the faulty ones too. However it is indeed a slow down and i will try to find another way to fix this issue. Thank you for your time.